src/tasks.js

```diff
@@ -0,0 +1,33 @@
+export default class Tasks {
+  constructor() {
+    this.arr = JSON.parse(localStorage.getItem('tasks')) || [];
+  }
+
+  setLocalStorageItem = () => {
+    localStorage.setItem('tasks', JSON.stringify(this.arr));
+  }
+
+  addNewTask = ({
+    description,
+    completed = false,
+    index = this.arr.length,
```

Comment:
- [ ] Kindly make sure the index of the first element will be `1` not `0`. It means if the fifth task is added must have an index that is equal to `5` not `4`. Please fix that in order to meet the project requirement. :wink: